lab_1_classify_by_unigrams/main.py

```diff
@@ -7,6 +7,14 @@
 
 
 def tokenize(text: str) -> list[str] | None:
+    if isinstance(text, str):
+        text = text.lower()
+        tokens = []
+        for i in text:
+            if i.isalpha():
+                tokens.append(i)
+        return tokens
+
     """
     Split a text into tokens.
```

Comment:
function body should be located after docstrings, not before

Reply:
done